listenbrainz/webserver/static/js/src/PlaybackControls.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from "react";
+import {
+  faEye,
+  faEyeSlash,
+  faFastBackward,
+  faFastForward,
+  faPauseCircle,
+  faPlayCircle,
+  faSortAmountDown,
+  faSortAmountUp,
+} from "@fortawesome/free-solid-svg-icons";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IconDefinition } from "@fortawesome/fontawesome-common-types"; // eslint-disable-line import/no-unresolved
```

Comment:
I'm not sure why this was getting triggered, typescript had no problems with the import and we have it installed in package.json